engines/wintermute/utils/path_util.cpp

```diff
@@ -74,6 +74,11 @@ AnsiString PathUtil::getDirectoryName(const AnsiString &path) {
 AnsiString PathUtil::getFileName(const AnsiString &path) {
 	AnsiString newPath = unifySeparators(path);
 	Common::String lastPart = Common::lastPathComponent(newPath, '/');
+
+	if (lastPart.empty()) {
+		return path;
+	}
```

Comment:
It is apparently not a WME thing:
https://bitbucket.org/MnemonicWME/wme1/src/62204728c14de49e49d5b6b70f93a00c310f2e0f/src/engine_core/wme_base/utils.cpp?at=default&fileviewer=file-view-default

Apparently, the whole getFilename method can be removed without damage, as in PR #737

Although: I didn't do a _lot_ of testing yet.